public/app/features/dashboard/components/Inspector/QueryInspector.tsx

```diff
@@ -39,9 +40,8 @@ interface State {
 }
 
 export class QueryInspector extends PureComponent<Props, State> {
-  formattedJson: any;
-  clipboard: any;
-  subscription?: Unsubscribable;
+  private formattedJson: any;
+  private subs = new Subscription();
```

Comment:
nit: shouldn't this initialization move into the constructor?

Reply:
yes, conflicted on what is best. This is technically the same as the constructor. The downside of moving it to the constructor is that I need to type it here and then duplicate it in the constructor. So this feels a bit leaner this way, sort of like declaring an instance value variable.